sim/paladin/protection/rotation.go

```diff
@@ -30,65 +30,56 @@ func (prot *ProtectionPaladin) customRotation(sim *core.Simulation) {
 	isExecutePhase := sim.IsExecutePhase20()
 
 	if prot.GCD.IsReady(sim) {
-	rotationLoop:
-		for _, spellNumber := range prot.RotationInput {
-			switch spellNumber {
-			case int32(proto.ProtectionPaladin_Rotation_JudgementOfWisdom):
-				if prot.JudgementOfWisdom.IsReady(sim) {
-					prot.JudgementOfWisdom.Cast(sim, target)
-					break rotationLoop
-				}
-			case int32(proto.ProtectionPaladin_Rotation_HammerOfWrath):
-				if isExecutePhase && prot.HammerOfWrath.IsReady(sim) {
-					prot.HammerOfWrath.Cast(sim, target)
-					break rotationLoop
-				}
-			case int32(proto.ProtectionPaladin_Rotation_Consecration):
-				if prot.Consecration.IsReady(sim) {
-					prot.Consecration.Cast(sim, target)
-					break rotationLoop
-				}
-			case int32(proto.ProtectionPaladin_Rotation_HolyWrath):
-				if prot.HolyWrath.IsReady(sim) {
-					prot.HolyWrath.Cast(sim, target)
-					break rotationLoop
-				}
-			case int32(proto.ProtectionPaladin_Rotation_Exorcism):
-				if prot.Exorcism.IsReady(sim) {
-					prot.Exorcism.Cast(sim, target)
-					break rotationLoop
-				}
-			case int32(proto.ProtectionPaladin_Rotation_ShieldOfRighteousness):
-				if prot.ShieldOfRighteousness.IsReady(sim) {
-					prot.ShieldOfRighteousness.Cast(sim, target)
-					break rotationLoop
-				}
-			case int32(proto.ProtectionPaladin_Rotation_AvengersShield):
-				if prot.AvengersShield.IsReady(sim) {
-					prot.AvengersShield.Cast(sim, target)
-					break rotationLoop
-				}
-			case int32(proto.ProtectionPaladin_Rotation_HammerOfTheRighteous):
-				if prot.HammerOfTheRighteous.IsReady(sim) {
-					prot.HammerOfTheRighteous.Cast(sim, target)
-					break rotationLoop
-				}
-			}
-		}
+	//rotationLoop:
+	
+		if isExecutePhase && prot.HammerOfWrath.IsReady(sim) {
+			// Always cast HoW if ready
+			prot.HammerOfWrath.Cast(sim, target)
+		} else
```

Comment:
Would you mind being consistent with the rest of our code, and keeping the `else` and `if` pieces on the same line?

Reply:
Done